src/components/Navigation.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react'
 import { Box, Link, List, Text } from '.'
 
 type NavigationProps = {
-  items: [string, FrontMatter[]][]
+  items: [string, any[]][]
```

Comment:
Not 100% sure but from what I understand, we are meant to try and avoid using `any`, right?
How come it is introduced here?

Reply:
Yeah just laziness on my part, will update with correct types now!